### PR TITLE
update doc [tvos]

### LINKED ISSUE #202: Apple TV support
i try build for tvos, but not find about it information

Create ticket this https://github.com/openfl/lime/issues/1005

@soywiz maybe you can find information about it for support in jtransc.

### PR DESCRIPTION
**Pull request by SergeyLabutin**
fix #202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +191,18 @@ Path to Android NDK []: C:\tools\android-ndk
 Path to Java JDK [C:\Program Files\Java\jdk1.8.0_131]:
 ```
 
+#### Setup AppleTV
+- Run `lime rebuild hxcpp tvos`
+- Remove stable lime `haxelib remove lime`
+- Install lime from GitHub `git clone --recursive https://github.com/openfl/lime`
```
Comment (soywiz) [inline README.md]: What is the reason for this? Do they have a fix in git verson after 5.2.1 that fixes tvos? I didnt see anything special related to that in their changelog

File: README.md [hunk 2]
```diff
@@ -163,7 +163,13 @@ So others could generate that without JVM and others could generate other target
 
 #### Installing JDK
 - Install [JDK8](http://www.oracle.com/technetwork/java/javase/downloads/)
+- Setup $JAVA_HOME
+```
+echo "export JAVA_HOME=$(/usr/libexec/java_home)" >> ~/.bash_profile
```
Comment (intrigus) [inline README.md]: This won't always work. So I'd leave it out because it's platform specific anyway.

Comment (soywiz) [inline README.md]: I use zsh on mac. Also for example intellij (and other programs) doesnt get those environments. In mac each version has different ways for creating actual "global" envs. Now im using one related to a plist file. It is a pity that there isnt an standarized way

File: README.md (hunk 1 above)
Comment (SergeyLabutin) [inline README.md]: From `haxelib install lime`, grab light version without all sources for rebuild.

Comment (SergeyLabutin) [inline README.md]: In base have not tvos, webos, tizen and many other platforms. tvos have many errors, i think other platform too. Maybe it's reason for none provide by default. Anyway tvos need support, apple love it.

File: README.md (hunk 2 above)
Comment (SergeyLabutin) [inline README.md]: add note about it

**Pull request merged**